src/selectors/markdownSelectors.js

```diff
@@ -1,5 +1,20 @@
 import marked from 'marked';
 
-export const getMarkdown = state => state.markdown;
+export const getMarkdownID = state => state.currentMarkdownID;
+
+export const getMarkdown = state => {
+  
+  let markdownBody;
+
+  state.markdownList.filter(markdown => {
+    if(markdown.id === getMarkdownID(state)) {
+      return markdownBody = markdown.body;
```

Comment:
this is how you would do it if you were using `forEach`.

instead:

```
const markdown = state.markdownList.find(.....);
return markdown?.body;
```

Reply:
Done.